entity-management-web/src/main/java/eu/europeana/entitymanagement/web/BaseRest.java

```diff
@@ -118,6 +118,10 @@ protected String serialize(
       EntityRecord entityRecord, FormatTypes format, List<EntityProfile> profiles)
       throws EuropeanaApiException {
 
+    if(profiles.contains(EntityProfile.dereference)) {
+      entityRecord.getEntity().dereference();
```

Comment:
dereference() method is likely to send a database request, therefore it must not be placed in the data object but in the service